aptos-move/framework/move-stdlib/sources/configs/features.move

```diff
@@ -287,13 +291,23 @@ module std::features {
     /// Whether enable TokenV2 collection creation and Fungible Asset creation
     /// to create higher throughput concurrent variants.
     /// Lifetime: transient
-    const CONCURRENT_ASSETS: u64 = 37;
+    const CONCURRENT_TOKEN_V2: u64 = 37;
```

Comment:
Is it okay to reuse the feature number 37 after deprecating CONCURRENT_ASSETS flag?

Reply:
yes, it's safe to rename constants. 

it is still the same flag (it was never enabled before), we just restrict it to gate less things (and so the rename)